installers/windows/update_path.ps1

```diff
@@ -0,0 +1,70 @@
+<#
+	.DESCRIPTION

```

Comment:
I'm not familiar with this format -  but shouldn't .DESCRIPTION be indented the same amount as .PARAMETER?

Reply:
It looks to be properly indented in my IDE -
![image](https://user-images.githubusercontent.com/6985763/58653915-6787c580-8334-11e9-94ea-9c7dc4f97349.png)